bin/run_qc.py

```diff
@@ -177,5 +183,37 @@ def detect_bad_cycle_tile_in_run(dataset, args):
             print('Bad tiles are: ' + ', '.join([str(c) for c in bad_tiles[lane]]))
 
 
+def get_all_project_gvcfs(project_folder):
+        gvcfs = []
+        for path in os.walk(project_folder):
+            gvcf = [i for i in path[2] if i.endswith('.g.vcf.gz')]
+            if gvcf:
+                gvcfs.append(os.path.join(path[0], ''.join(gvcf)))
+        return gvcfs
+
+def peddy(dataset, args):
+    all_gvcfs = []
+    sample_ids = []
+    if args.samples:
+        cfg.merge(cfg['sample'])
+        for sample_id in args.samples:
+            sample_ids.append(sample_id)
+            s = get_documents(sample_id)[0]
+            project_id = s.get('project_id')
+            gvcf_file = s.get('user_sample_id') + '.g.vcf.gz'
+            gvcf = util.find_file(cfg['input_dir'], project_id, sample_id, gvcf_file)
+            all_gvcfs.append(gvcf)
+    elif args.projects:
+        cfg.merge(cfg['project'])
+        for project_id in args.projects:
+            samples_for_project = clarity.get_sample_names_from_project(project_id)
+            sample_ids.extend(samples_for_project)
+            project_folder = util.find_file(cfg['input_dir'], project_id)
+            all_gvcfs.extend(get_all_project_gvcfs(project_folder))
+
+    genotyped_gvcfs, gatk_outfile = qc.Genotype_gVCFs(all_gvcfs)
```

Comment:
This will not work, as it will try to construct a Genotype_gVCFs object with a list of files and unpack that into two variables. Instead, I would construct a small pipeline here, and use `all_gvcfs` to construct a Genotype_gVCFs stage object to pass into Peddy via previous_stages.